src/app/feed/feed-card/feed-card.component.html

```diff
@@ -1,7 +1,7 @@
 <a class="post-link" target="_blank">
 	<div class="card" (click)="showLightBox.emit({feedIndex : feedIndex , show : 'show'})" in-viewport (inViewport)="inview($event)">
 		<div class="card-image">
-			<a (click)="showHideLightbox(profileURL)">
+			<a (click)="showHideLightbox(profileURL)" class="openProfile">
```

Comment:
Do we explicitly need to add this to so that we can add test for that?

Reply:
No, not really. I'll just fix this :sweat_smile: